templates/travel/elements/_btn.css

```diff
@@ -0,0 +1,50 @@
+/**
+ * Copyright 2017 The AMP Start Authors. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS-IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+/* Override properties from ampstart-button. */
+
+ .ampstart-btn {
```

Comment:
<img width="570" alt="screen shot 2017-10-24 at 2 13 21 pm" src="https://user-images.githubusercontent.com/1037311/31968401-d4bf14ae-b8c5-11e7-810a-964675a09dae.png">

Reply:
Please remove all redundant stuff, also when you push commits, it deletes all the comments i gave , it would be nice to see the comments that have not been addressed, not sure how github deals with this .